client/manager_sql.go

```diff
@@ -1,18 +1,31 @@
 package client
 
 import (
-	"encoding/json"
+	"fmt"
 	"github.com/imdario/mergo"
 	"github.com/jmoiron/sqlx"
 	"github.com/ory-am/fosite"
 	"github.com/pkg/errors"
+	"strings"
 )
 
 var sqlSchema = []string{
 	`CREATE TABLE IF NOT EXISTS hydra_client (
-	id      varchar(255) NOT NULL PRIMARY KEY,
-	version int NOT NULL DEFAULT 0,
-	client  json NOT NULL)`,
+	id      	varchar(255) NOT NULL PRIMARY KEY,
+	client_name  	text NOT NULL,
```

Comment:
Remember NOT NULL on not indexed fields is expensive (not important here, since there isn't a lot of clients)

Are all of the fields really required?